demos/serverless/src/handlers.js

```diff
@@ -208,3 +208,55 @@ exports.end = async(event, context, callback) => {
   }).promise();
   callback(null, response);
 };
+
+exports.logs = async (event, context) => {
+  var sequenceToken = "";
+  const logGroupName = "BrowserLogs";
```

Comment:
If we want to make cloud watch logging the part of official demo, we need to move log group creation to the CF template, and pass its name as an argument to the lambda